manifests/service.pp

```diff
@@ -42,6 +47,10 @@
     fail('$service_name must be specified when $use_init is set to true')
   }
 
+  if $service_enable != undef and ! $use_init {
+    fail('$use_init must be set to true when $service_enable is set')
```

Comment:
Hm. What would happen if we allowed service to be enabled when there is not init script?